nixos/modules/services/web-apps/tt-rss.nix

```diff
@@ -631,9 +631,12 @@ let
         serviceConfig = {
           User = "${cfg.user}";
           Group = "tt_rss";
-          ExecStart = "${pkgs.php}/bin/php ${cfg.root}/update.php --daemon";
+          ExecStart = "${pkgs.php}/bin/php ${cfg.root}/update.php --daemon --quiet";
           StandardOutput = "syslog";
           StandardError = "syslog";
+          Restart = "on-failure";
+          RestartSec = "60";
+          SyslogIdentifier = "tt-rss";
```

Comment:
@symphorien see https://github.com/NixOS/nixpkgs/issues/86797#issuecomment-629872567 and let me know what you think :thinking:

Reply:
For tt-rss it is because otherwise it is logged as php, not tt-rss, so this is orthogonal to the naming of start scripts.

Reply:
Ah, yeah that makes sense. Cool, thanks! :+1: